src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -149,18 +158,18 @@ public void WriteBindings(IResource resource, bool emitContainerPort = false)
     /// TODO: Doc Comments
     /// </summary>
     /// <param name="resource"></param>
-    public void WriteEnvironmentVariables(IResource resource)
+    public async Task WriteEnvironmentVariables(IResource resource)
```

Comment:
WriteEnvironmentVariablesAsync

Cancellation token

Reply:
It's on the context